### ISSUE TITLE
Show Rufo error, when failing to format

### ISSUE THREAD

**Issue by asteinarson** [jnbt/vscode-rufo#8]
This is really a feature request. Not too complex I believe, but it would be quite useful. 

If I have a Ruby file with a syntax error in it, like _test.rb_: 
```
module Arst
  def my_func
    puts "my_func"
  end
```

Then, trying to format I get the error _Rufo failed with exit code: 1_. That doesn't help me very much. 

However, running Rufo from the terminal I see what (and exactly where) the error is:
```
$ rufo test.rb
test.rb:11 syntax error, unexpected end-of-input, expecting `end'
```

I would like to get this error message shown inside VsCode (popup, simple dialog, or expandable, in the red message now saying _Rufo failed with exit code: 1_).

If I could get an underline, on this line number, in the editor.... that would be icing on the cake.

**Comment by jnbt** [comment]
@asteinarson I did some quick work and was able to manage the following solution in PR #11

![Bildschirmfoto 2020-12-21 um 12 00 35](https://user-images.githubusercontent.com/678542/102770278-275bbe00-4384-11eb-9fff-f9ae5bd0dcf7.png)

I also would love the idea to underline the error in the editor. Sadly this would be more work, especially to circumvent wrong lines highlighted when only formatting a selection (and not the whole document).

**Comment by jnbt** [comment]
Released as version 0.0.5
